src/java/picard/util/IntervalListTools.java

```diff
@@ -32,9 +32,27 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = " General tool for manipulating interval lists, " +
-                "including sorting, merging, padding, uniqueifying, and other set-theoretic operations. Default operation if given one or more inputs is to " +
-                "merge and sort them.  Other options are controlled by arguments.",
+        usage = "General tool for manipulating interval lists, including sorting, merging, padding, customizing, " +
+                "and other set-theoretic operations. Default operation if given one or more inputs is to merge" +
+                " and sort them.  Other options are controlled by arguments.<br /><br />" +
+
+                "Lists intervals with respect to a reference sequence that can be written to and read from a file." +
+                " The file format is relatively simple and reflects the SAM alignment format to a degree." +
+                "  A SAM style header must be present in the file, which lists the sequence records against" +
```

Comment:
I'm not sure this is universally true -- I often use Picard interval lists that don't have a header. The header may be required for this or other tools, but afaik it is not a requirement of the format in general.